sim/midas/src/main/scala/midas/widgets/Lib.scala

```diff
@@ -208,7 +208,7 @@ class MCRFileMap() {
   private val regList = ArrayBuffer[MCRMapEntry]()
 
   def allocate(entry: MCRMapEntry): Int = {
-    Predef.assert(!name2addr.contains(entry.name), "name already allocated")
+    Predef.assert(!name2addr.contains(entry.name), s"name already allocated '${entry.name}'")
```

Comment:
You ran into this eh. Where was the collision?

Reply:
I think it was because we had coverpoints with the same label in the same module.